plugins/extraction/g_extraction.mlg

```diff
@@ -60,6 +60,8 @@ let pr_language = function
   | Scheme -> str "Scheme"
   | JSON -> str "JSON"
 
+let access = Library.indirect_accessor[@@warning "-3"]
```

Comment:
This is not very satisfying IMO, I'll probably wait for a better solution before doing overlays (probably after https://github.com/coq/coq/pull/18421 is merged)

Reply:
I'm OK with this, but indeed it'd be good to head towards a more principled solution.

Reply:
TBH I don't want to do this for the overlays so this PR is waiting for the more principled solution.